### ISSUE TITLE
-finetune_model [PATH TO CONTINUOUS MODEL]

### ISSUE THREAD

**Issue by Pink-Dolphin** [haiciyang/LaDiffCodec#4]
Should we train the Discrete autoencoder or the Continuous autoencoder first? If we train the Continuous autoencoder first, So what does -finetune_model [PATH TO CONTINUOUS MODEL] mean

**Comment by haiciyang** [comment]
Thank you for the question. These two models are trained separately. So it doesn't matter which is trained first. This argument just provides an option to finetune from the pre-trained checkpoints. Thanks

**Comment by Pink-Dolphin** [comment]
Thank you very much for answering my question. In addition, I would like to ask that when I was training the diffusion model, I encountered this problem: x = torch.cat((x_cond, x), dim=1)
RuntimeError:  Sizes of tensors must match except in dimension 1. Expected size 4800 but got size 1200 for tensor number 1 in the list. Perhaps the upsampling_ratios[5,4,2] set in unet changed x_cond from (5, 128, 120) to (5, 128, 4800) and x to (5, 128, 1200). How do I solve this? Can I use upsampling_ratios[5,2]
![PixPin_2024-11-29_09-47-58](https://github.com/user-attachments/assets/c9afbd29-c43a-494d-ac74-d74f32fc1978)

**Comment by haiciyang** [comment]
Yes you can use arbitrary upsampling_ratios.  You just need to give it to the training arguments. For example --upsampling_ratios 5 2
